core/tests/protractor/accessibility.js

```diff
@@ -41,6 +94,51 @@ describe('screenreader and keyboard user accessibility features', function() {
         .getAttribute('id'));
   });
 
+  it('should test navigation shortcuts for the community-library page',
+    async function() {
+      await testNavigationShortcuts('community-library');
+    });
+
+  it('should test navigation shortcuts for the creator-dashboard page',
+    async function() {
+      await testNavigationShortcuts('creator-dashboard');
+    });
+
+  it('should test navigation shortcuts for the get-started page',
+    async function() {
+      await testNavigationShortcuts('get-started');
+    });
+
+  it('should test navigation shortcuts for the about page',
+    async function() {
+      await testNavigationShortcuts('about');
+    });
+
+  it('should test navigation shortcuts for the privacy-policy page',
+    async function() {
+      await testNavigationShortcuts('privacy-policy');
+    });
+
+  it('should test navigation shortcuts for the donate page',
+    async function() {
+      await testNavigationShortcuts('donate');
+    });
+
+  it('should test navigation shortcuts for the donate preferences page',
+    async function() {
+      await testNavigationShortcuts('preferences');
+    });
+
+  it('should test navigation shortcuts for the donate learner-dashboard page',
+    async function() {
+      await testNavigationShortcuts('learner-dashboard');
+    });
+
+  it('should test navigation shortcuts for the notifications page',
+    async function() {
+      await testNavigationShortcuts('notifications');
+    });
```

Comment:
Sorry if I wasn't clear, but what I meant by [this](https://github.com/oppia/oppia/pull/10036#discussion_r460455320) comment was to get rid of the `testNavigationShortcuts` function as it is obscuring what actions are being taken and what assertions are being made. In each unit test you can repeat the same lines, e.g. for the get started test:

```
await browser.get('some url not related to the url under test, maybe the home page?');
await waitForUrlRedirection('some url not related to the url under test, maybe the home page?');

await triggerKeys('0');
await waitForUrlRedirection(getStartedUrl);

expect(await browser.getCurrentUrl()).toEqual(getStartedUrl);
```

In test code, it's okay to have some code duplication for readability/clarity.

Reply:
Done